### ISSUE TITLE
crashes reliably when using successive 'find in files' searches

### ISSUE THREAD

**Issue by ianthehawk** [notepad-plus-plus/notepad-plus-plus#12181]
### Description of the Issue
Notepad++ 8.4 crashes reliably when using successive 'find in files' searches

### Steps to Reproduce the Issue
1. open a file
2. double-click a word to select it
3. ctrl-shift-f to search for the word in a group of files
4. enter. This produces the expected result
5. double-click one of the items found
6. double-click another word in the newly opened file
7. ctrl-shift-f 
8. enter now causes a crash

### Expected Behavior
no crash
### Actual Behavior
crash

### Debug Information
Notepad++ v8.4.1   (64-bit)
Build time : May  8 2022 - 19:51:18
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 10 Home (64-bit) 
OS Version : 2009
OS Build : 19044.2006
Current ANSI codepage : 1252
Plugins : mimeTools.dll NppConverter.dll NppExport.dll 

I have tried using versions 8.4.5 8.4.4 8.4.1 with same result. Version 8.3.3 does not have the problem.

**Comment by alankilborn** [comment]
I could obtain no crash in attempting to reproduce the problem.

@ianthehawk Can you take the `license.txt` file in the directory where notepad++.exe is, and search that directory for `*.txt` files and see if you can reproduce the problem?  If you can, then you can list the details from that and someone can try replicating more of an exact situation that is failing for you.

**Comment by ianthehawk** [comment]
After posting the report I had reinstalled 8.3.3 to continue working. When you commented I reinstalled 8.4.4, tried what you suggested and the problem did not occur!!!

Weirdly the problem no longer occurs when operating on my own files! Given that before posting the bug report I had tried it on 3 different versions of 8.4, when I reverted to 8.3 the problem went away and when I reinstalled 8.4 it reappeared it is a bit of a mystery. The only explanation I can come up with is that changing the Directory field to try your test cleared the problem, which suggests the config data had got corrupted, either due to a bug or some random event in the computer.

If it helps, I was a bit vague about the "crash". What actually happened was that the UI froze. npp did not terminate, and did not appear to be in a loop, but it could only be closed from the taskbar. 

I will continue to use 8.4.5 and hope it does not recur.
